packages/elements/src/stores/catalog.ts

```diff
@@ -72,7 +72,7 @@ const createCatalogStore = () => {
     const { data, errors } = await client.clinical.catalog.getCatalogs({
       fragment: CatalogFieldsMap
     });
-    setStore('catalogs', {
+    return setStore('catalogs', {
```

Comment:
Whats the deal here?

Reply:
ah... I could have swore this was giving me an error, I'm going crazy reverted